index.js

```diff
@@ -1,55 +1,17 @@
 const express = require('express');
 const cors = require('cors');
-const { ApolloServer, gql } = require('apollo-server-express');
 const PORT = process.env.PORT || 5000
 
-// A schema is a collection of type definitions (hence "typeDefs")
-// that together define the "shape" of queries that are executed against
-// your data.
-const typeDefs = gql`
-  # Comments in GraphQL strings (such as this one) start with the hash (#) symbol.
+const { ApolloServer } = require('apollo-server-express');
+const typeDefs = require('./src/graphql/typeDefs/index');
```

Comment:
When you reference index file, you don't need to specify it.
const typeDefs = require('./src/graphql/typeDefs');